/*
 * color_filter_calibration.hpp
 *
 *  Created on: Mar 19, 2014
 *      Author: Olivier BALLAND
 */

#ifndef COLOR_FILTER_CALIBRATION_HPP
#define COLOR_FILTER_CALIBRATION_HPP


// Thresholds
int H_MIN,H_MAX,S_MIN,S_MAX,V_MIN,V_MAX;

// Robot selection
std::string nao;


#endif /* COLOR_FILTER_CALIBRATION_HPP */
